src/components/addTranslation/AddTranslation.tsx

```diff
@@ -31,8 +34,10 @@ const AddTranslation = ({ setShowModal }: AddTranslationState) => {
     <div className="add-translation">
       <Formik
         initialValues={{
-          name: "",
+          title: "",
           author: "",
+          isEmail: "",
```

Comment:
It works, but it'd be better to set initial value to false or true as it's boolean.

Reply:
Tried to,  it's tricky here. Normally you write down < input... value="false">, but if it's given from outside it's treating it like a string "`false`" and from it we have true ;)